Parsers/Logstash/input-aws_s3-output-loganalytics.conf

```diff
@@ -0,0 +1,34 @@
+// Logstash configuration file to ingest custome json file
+// Last Updated Date: Nov 5, 2019
+//
+// Usage Instructions:
+// Follow the wiki article for installation and usage Instructions.
+// https://github.com/Azure/Azure-Sentinel/wiki/Ingest-Custom-Logs-LogStash
+//
+// Parser Notes:
+// 1. Make sure the log type name is specified as alpha characters- no digits or special characters allowed.
+//
+input{
+s3 {
+access_key_id => "<AWS_ACCESS_KEY_ID>"
+secret_access_key => "<AWS_SECRET_ACCESS_KEY>"
+bucket => "<AWS_S3_BUCKET_NAME>"
+codec => "json"
+}
```

Comment:
Seems like structurally, these should have tabs or indents in them from a code readability perspective.

Reply:
Good point. Added indents